src/GenFacades/Program.cs

```diff
@@ -0,0 +1,714 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Cci;
+using Microsoft.Cci.Extensions;
+using Microsoft.Cci.MutableCodeModel;
+using Microsoft.Fx.CommandLine;
+
+namespace GenFacades
+{
+    public class Program
+    {
+        private const uint ReferenceAssemblyFlag = 0x70;
+
+        public static void Main(string[] args)
+        {
+            string seeds = null;
+            string contracts = null;
+            string facadePath = null;
+            Version assemblyFileVersion = null;
+            bool clearBuildAndRevision = false;
+            bool ignoreMissingTypes = false;
+            bool buildDesignTimeFacades = false;
+            string inclusionContracts = null;
+            ErrorTreatment seedLoadErrorTreatment = ErrorTreatment.Default;
+            ErrorTreatment contractLoadErrorTreatment = ErrorTreatment.Default;
+            string[] seedTypePreferencesUnsplit = null;
+            bool forceZeroVersionSeeds = false;
+            string partialFacadeAssemblyPath = null;
+
+#if COREFX
+            string commandLine = "GenFacades.exe " + string.Join(" ", args); // CommandLine parser assumes first arg is exe name.
+#else
+            string commandLine = Environment.CommandLine;
```

Comment:
It looks like there is already one, I called it out with another comment. I would switch both cases to using that here.